config/samples/infinispan_v1_infinispan.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infinispan.org/v1
```

Comment:
I assume the plan is to replace these with the `deploy/crds/*_cr.yaml` fields to this dir? and remove `deploy/`?

Reply:
`deploy` dir needs to be removed. I would keep the CRD generated by the new SDK, after a verification of course